apps/client/src/components/record-state/record-state.tsx

```diff
@@ -1,14 +1,16 @@
-import { useState } from 'react';
 import styles from './record-state.module.css';
+import { useGetToastInPeriodQuery, useGetRecordPeiodQuery } from '../../store';
 
 export const RecordState: React.FC = () => {
-  const [recordToasts, setRecordToasts] = useState(0);
-  const [currentToasts, setCurrentToasts] = useState(0);
+  const { data: periodToasts } = useGetToastInPeriodQuery();
+  const { data: recordToasts } = useGetRecordPeiodQuery();
 
   return (
     <button className={styles.record}>
       <div className={styles.recordNum}>
-        {currentToasts} / {recordToasts}
+        {recordToasts && periodToasts
+          ? `${periodToasts} / ${recordToasts}`
+          : '0/0'}
```

Comment:
Consider Changing `0/0` to something else  like `none/none` for example.
0/0 is an actual possible occurance but we want to differentiate it from a case where the data is problomatic